Gemfile

```diff
@@ -21,6 +21,7 @@ end
 manageiq_plugin "manageiq-schema"
 
 # Unmodified gems
+gem "addressable",                      "~>2.5",            :require => false
```

Comment:
We already require addressable in core code:

```
% git grep Addressable
app/models/service_template_transformation_plan_task.rb:        :path     => "/vmfs/volumes/#{Addressable::URI.escape(storage.name)}/#{Addressable::URI.escape(source.location)}"
```

We already have dependencies that require addressable 2.5 at a minumum:

```
    manageiq-providers-scvmm (0.1.0)
      addressable (~> 2.4)
    google-api-client (0.50.0)
      addressable (~> 2.5, >= 2.5.1)
    azure-armrest (0.10.0)
      addressable (~> 2.5.0)
    azure-signature (0.3.0)
      addressable (~> 2)
    hawkular-client (5.0.0)
      addressable
    http (4.4.1)
      addressable (~> 2.3)
    json-schema (2.8.1)
      addressable (>= 2.4)
    signet (0.14.1)
      addressable (~> 2.3)
    wasabi (3.6.1)
      addressable
    webmock (3.11.2)
      addressable (>= 2.3.6)
```